product_variant_default_code/tests/test_variant_default_code.py

```diff
@@ -64,7 +64,7 @@ def test_01_check_default_codes(self):
             self.assertEqual(product.default_code, expected_code)
 
     def test_02_check_default_codes_preexistent_mask(self):
```

Comment:
This test should pass in manual mode, but now it doesn't:
```
AssertionError: u'L-Gr' != u'P01/LGr'
```
Can you figure out why?